superset/config.py

```diff
@@ -409,12 +410,13 @@ def _try_json_readsha(filepath: str, length: int) -> Optional[str]:
     # This could cause the server to run out of memory or compute.
     "ALLOW_FULL_CSV_EXPORT": False,
     "UX_BETA": False,
+    "SCREENSHOTS_USE_RETINA_HIRES": False
```

Comment:
this is the only change.. the rest is automated linting.

Reply:
Nit, trailing comma:

```suggestion
    "SCREENSHOTS_USE_RETINA_HIRES": False,
```